csharp-code-evaluator-ut/csharp-script-ut.csproj

```diff
@@ -36,7 +36,7 @@
   <ItemGroup>
     <Reference Include="nunit.framework, Version=2.6.3.13283, Culture=neutral, PublicKeyToken=96d09a1eb7f44a77, processorArchitecture=MSIL">
       <SpecificVersion>False</SpecificVersion>
-      <HintPath>..\dependencies\nunit.framework.dll</HintPath>
+      <HintPath>C:\Program Files (x86)\NUnit 2.6.2\bin\framework\nunit.framework.dll</HintPath>
```

Comment:
@jsbattig I re-clone the repo and I was able to pull the dll.  I pushed new changes that doesn't have this code change, but GIT is not removing it from here.